src/helm/benchmark/metrics/summarization_critique_metrics.py

```diff
@@ -23,30 +23,34 @@ class SummarizationCritiqueMetric(Metric):
 
     def __init__(self, num_respondents: int) -> None:
         self._template = CritiqueTaskTemplate(
-            name="HELM Summarization Evaluation",
+            name="summarization_critique",
```

Comment:
Do we need to include the full run spec and maybe even the model name in `name`? Otherwise, would the output template and data files get overwritten if we're evaluating multiple models or even scenarios?

Reply:
A single execution of `helm-run` with multiple models and run specs will result with only a single output layout and CSV file. This is because we maintain a singleton exporter per template.

It was previously true that multiple invocations of `helm-run` would result in the files getting overwritten. I updated the PR to fix this by appending a unique(-ish) timestamp to the generated file.